packages/graphql/mock-server/mixin.core.js

```diff
@@ -57,6 +57,7 @@ module.exports = class GraphQLMockServerMixin extends Mixin {
 
   async fetcher({ query, variables, operationName, url, headers }) {
     const fetchResult = await fetch(url, {
+      credentials: 'include',
```

Comment:
Why have you added `credentials` here? Isn't this executed on the server where there are no cookies?

Reply:
This has been added by Mike in my node modules folder. But you're right this makes no sense to me either at the moment. It's not needed for the current mock server use case anyway, so I've removed it for now.